### PR TITLE
Fixes missed  entity on scan

### LINKED ISSUE #31: Nitro Scam Missed
**Describe the bug**
When nitro is excluded, tghe bot completely ignored

**To Reproduce**
1. Send a message like this:
>  `@`  everyone
> Wh\u043e is first? :)
> https://discerd.gift/FwrVcACacD
2. Observe the bot ignores it

**Expected behavior**
Bot chatches and bans the member.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows

**Additional context**

Thanks to @CasparN for catching it

### PR DESCRIPTION
**Pull request by Bluesy1**
Closes #31

### CODE REVIEW

**Comment by Bluesy1** [comment]
Also starts working towards test coverage of bot to confirm things work as expected

**Comment by Bluesy1** [comment]
I plan to add more tests before i mege this, but i still want you to review the changes so far Caspar

**Comment by Bluesy1** [comment]
I added a few more tests, i can only test ignore conditions though, without getting into creating fake event loops

**Comment by Bluesy1** [comment]
At this point, it *should* be ready for a review.

File: extensions/config.py [hunk 1]
```diff
@@ -181,21 +179,119 @@ async def config_moderation(ctx: lightbulb.Context):
             f"{'**CHANGED** ' if ctx.options.update_log.id != result['updates'] else ''}Bot Updates Channel",
             # pylint: disable=line-too-long
             f"<#{ctx.options.update_log.id}>", inline=True)
-    await ctx.respond(embed=embed, flags=EPHEMERAL)
+    await ctx.respond(embed=embed)
     await mydb.close()
 
 
-@CONFIG.command()
+@config.child
```
Comment (Bluesy1) [inline extensions/config.py:186]:
```suggestion
@config.child
```
Just realized, this also pushes the initial testing for user inputted ban messages.

File: extensions/reports.py [hunk 2]
```diff
@@ -70,14 +72,14 @@ async def timeout_callback(self) -> None:
         """Timeout Callback"""
         if self.unresolved:
             try:
-                await (await report_commands.app.rest.fetch_user(self.author_id)).send(
+                await (await REPORT_COMMANDS.app.rest.fetch_user(self.author_id)).send(
                     f"Report for message with the following content has time out. please feel free to resubmit if you "
```
Comment (CasparN) [inline extensions/reports.py:76]:
```suggestion
                    f"Report for message with the following content has timed out. please feel free to resubmit if you "
```

**Review by CasparN — CHANGES_REQUESTED** [review]
Seems pretty good to me! 1 thing that should be fixed but besides that its pretty epic

**Review by CasparN — APPROVED** [review]
👏

**Pull request merged**